### ISSUE TITLE
Better handling of account failure

### ISSUE THREAD

**Issue by phareous** [Memmy-App/memmy#595]
**Is your feature request related to a problem? Please describe.**

If an account is no longer authenticated, feeds silently fail. Also, even when successful, feeds take many seconds to update with no feedback

**Describe the solution you'd like**

Show a spinning icon or other indicator while a feed is refreshing

When a feed cannot be refreshed or displayed, present an error saying so

**Comment by homer1993** [comment]
First of all I would like to express my thanks for the app and the effort. Lemmy is relatively new to me and having the app in the iOS store was a game changer for me. The mobile experience of lemmy has improved in an instant and I'm very thankful that the app even exists!

My feed didn't load all the time even though the app said my login was working. It wasn't until I deleted the app and reconfigured it that I got the message that it was not possible to login with the email address. I think better account management can help here.
